src/components/metadata/index.js

```diff
@@ -55,6 +57,20 @@ const MetaData = ({ visible = true, metadata, onToggleNodeSelected }) => {
       </div>
       <dl className="pipeline-metadata__list">
         <MetaDataRow label="Type:" value={metadata.node.type} />
+        <MetaDataRow
+          label="Dataset Type:"
+          visible={isDataNode}
+          kind="type"
+          value={metadata.datasetType}
+        />
+        <MetaDataRow label="File Path:" kind="path" value={metadata.filepath} />
+        <MetaDataRow
+          label={`Parameters (${metadata.parameters?.length || '-'}):`}
+          visible={isParametersNode || isTaskNode}
+          commas={false}
+          inline={false}
+          value={metadata.parameters}
+        />
```

Comment:
Design question: Are we better off showing these fields regardless of whether they're populated, or only showing them if populated?
![image](https://user-images.githubusercontent.com/1155816/101910509-eced6680-3bb6-11eb-85bf-8c33849aef31.png)

Reply:
Yeah it's a good point and I did this intentionally, as I think it's important to show empty values in technical use cases like this. We can discuss and improve it later though if needed I think.

Reply:
Sweet as! cc @GabrielComymQB let's come back to this in 2021